src/components/List.jsx

```diff
@@ -0,0 +1,59 @@
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import { updateTodo } from "../features/todo";
+import { TiTick } from "react-icons/ti";
+import { AiTwotoneDelete, AiTwotoneEdit } from "react-icons/ai";
+const List = ({ item, handleChecked, handleDelete, index }) => {
+  const [editInput, setEditInput] = useState("");
+  const [isClicked, setIsClicked] = useState(false);
+
+  const handleChange = (event) => {
+    setEditInput(event.target.value);
+  };
```

Comment:
Since this function has a one-line functionality, you can remove the curly braces.